common/reflect_resource.go

```diff
@@ -74,6 +218,111 @@ func MustSchemaPath(s map[string]*schema.Schema, path ...string) *schema.Schema
 	return sch
 }
 
```

Comment:
Will need to figure out a way to merge these with the code introduced in https://github.com/databricks/terraform-provider-databricks/pull/3044. Shouldn't be too hard